SimPEG/EM/TDEM/ProblemTDEM.py

```diff
@@ -1126,3 +1191,10 @@ def getRHS(self, tInd):
     def getRHSDeriv(self, tInd, src, v, adjoint=False):
         return Utils.Zero()  # assumes no derivs on sources
 
+    def getAdc(self):
```

Comment:
Here we are assuming dirichlet b.c., so when considering air-earth interface we need to put air cells.
I guess that's fine for TDEM.